cerberus/kubernetes/client.py

```diff
@@ -181,3 +182,15 @@ def monitor_cluster_operator(cluster_operators):
     else:
         status = True
     return status, failed_operators
+
+
+# See if api server is available
+def is_api_server_available(url):
```

Comment:
This function can be made generic in the sense that we can check if any of the provided url's is reachable. This will be helpful in solving https://github.com/openshift-scale/cerberus/issues/48 as well. Thoughts?

Reply:
Yes I totally agree on this one. I had thought about it when I started the issue #48 but wasn't sure if I should change it in the next issue